src/features/market-status/components/MarketStatus.tsx

```diff
@@ -23,7 +25,10 @@ export const MarketStatus: FC<MarketStatusProps> = ({ stock, keyStats }) => {
           Real-Time Price as of {correctFormat} EST
         </span>{" "}
         {keyStats && (
-          <span className={marketSign(keyStats.open)}> {marketStatus}</span>
+          <span className={marketSign(keyStats.isUSMarketOpen)}>
+            {" "}
```

Comment:
Q: Is it not better to use CSS to create spacing between the` <span>` and `{marketStatus}` items as opposed to `{" "}`?

Reply:
Good point, will change it